demos/index.html

```diff
@@ -25,9 +28,37 @@
     </head>
 
     <body>
+
+        <div id="floec-overviewPanel" class="fl-overviewPanel-container"></div>
+
         <div id="floec-chartAuthoring"></div>
 
         <script type="text/javascript">
+            fluid.overviewPanel("#floec-overviewPanel", {
+                resources: {
+                    template: {
+                        href: "../src/lib/infusion/src/components/overviewPanel/html/overviewPanelTemplate.html"
+                    }
+                },
+                strings: {
+                    componentName: "Chart Authoring Tool",
+                    feedbackText: "Have a comment?",
+                    feedbackLinkText: "Email us"
```

Comment:
It's actually because I used the overviewPanel demo as my model for implementation, and it has those configurable strings - I'll remove them.